ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/nested-virtualization/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-workflow.yaml

```diff
@@ -4,11 +4,15 @@ workflow:
     pre:
     - chain: cucushift-installer-rehearse-gcp-ipi-customer-vpc-provision
     - ref: cucushift-installer-check-gcp-nested-virtualization
+    - ref: cucushift-installer-check-gcp-disktype-compute
```

Comment:
Shall we rename the install workflow name accordingly to indicate `disk-type-size`